src/main/java/mate/academy/bookstore/model/Category.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@Table(name = "categories")
+@Data
+@NoArgsConstructor
+@SQLDelete(sql = "UPDATE categories SET is_deleted = true WHERE id = ?")
+@SQLRestriction("is_deleted = false")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    private String description;
+
+    @Column(nullable = false)
+    private boolean isDeleted = false;
```

Comment:
```suggestion
    private boolean isDeleted;
```
It’s  default value for boolean type